apps/dashboard/features/attack-profitability/hooks/useTreasuryAssetNonDaoToken.ts

```diff
@@ -3,25 +3,29 @@ import { BACKEND_ENDPOINT } from "@/shared/utils/server-utils";
 import { DaoIdEnum } from "@/shared/types/daos";
 import useSWR, { SWRConfiguration } from "swr";
 import axios from "axios";
-export interface TreasuryAssetNonDaoToken {
-  date: string;
-  totalAssets: string;
+
+export interface TreasuryAssetData {
+  date: number;
+  liquidTreasury: number;
 }
 
-export const fetchTreasuryAssetNonDaoToken = async ({
+export const fetchTreasuryAssetData = async ({
   daoId,
   days,
+  order = "asc",
 }: {
   daoId: DaoIdEnum;
   days: string;
-}): Promise<TreasuryAssetNonDaoToken[]> => {
+  order?: "asc" | "desc";
+}): Promise<TreasuryAssetData[]> => {
   const query = `
-  query getTotalAssets {
-  totalAssets(days:_${days}){
-    totalAssets
-    date
-  }
-}`;
+  query {
+    totalAssets(days: _${days}, order: ${order}) {
+      date
+      liquidTreasury
```

Comment:
this change is breaking the whole application when this new field isn't found.

could you handle the error response in a way that it doesn't happen?